gossip/emitter/emitter_test.go

```diff
@@ -115,4 +122,17 @@ func TestEmitter(t *testing.T) {
 	t.Run("tick", func(t *testing.T) {
 		em.tick()
 	})
+
+	t.Run("chooseParentsGenesis", func(t *testing.T) {
+		selfParent, parents, ok := em.chooseParents(idx.Epoch(1), cfg.Validator.ID)
+		if selfParent != nil {
+			t.Error("genesis event must not have self parent")
+		}
+		if len(parents) > 0 {
+			t.Error("genesis event must not have any parents")
+		}
+		if !ok {
+			t.Error("genesis parent assignment must succeed without any parents")
+		}
+	})
```

Comment:
This is a rather unexpected location for a test that targets a function implemented in `parents.go`. By the usual convention, it should be placed in a file named `parents_test.go`.

The test also seems to be rather unrelated to any other test in this file, so factoring it out should be rather straight forward.